languages/c/code/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(codecrafters-{{course_slug}})
+
+file(GLOB_RECURSE SOURCE_FILES app/*.c app/*.h)
+
+set(CMAKE_C_STANDARD 23) # Enable the C23 standard
+
+add_executable(codecrafters-build-{{course_slug}}-c ${SOURCE_FILES})
```

@andy1li I think this can be simplified too. Since this is local the repository, we could just use: 

```suggestion
add_executable({{course_slug}} ${SOURCE_FILES})
```

---

The reason we have `codecrafters-build-<slug>-<lang>` previously is for cases where the executable is stored in `/tmp` (and shouldn't collide with other codecrafters projects).